File: main (2).cpp
/******************************************************************************

                              Online C++ Compiler.
               Code, Compile, Run and Debug C++ program online.
Write your code in this editor and press "Run" button to compile and execute it.

*******************************************************************************/

#include <iostream>
#include <bits/stdc++.h>

using namespace std;
int main()
{
    int n = 50, pos = 3;
    int arr[50] = {1,2,3,4,5};
    for(int i=5; i>=pos; i--){
        arr[i] = arr[i-1];
    }
    arr[pos] = n;
    for(int i=0; i<6; i++){
        cout<<arr[i];
    }
        return 0;
}

File: main (2).py
'''

Welcome to GDB Online.
GDB online is an online compiler and debugger tool for C, C++, Python, Java, PHP, Ruby, Perl,
C#, OCaml, VB, Swift, Pascal, Fortran, Haskell, Objective-C, Assembly, HTML, CSS, JS, SQLite, Prolog.
Code, Compile, Run and Debug online from anywhere in world.

'''


s = 'swaaeooypniil'
lis = []
for i in s:
    if i == 'a' or i =='e' or i=='i' or i == 'o' or i == 'u' or i == 'y':
        if i in lis:
            pass
        else:
            lis.append(i)
print(lis)
